bufstream/iceberg-quickstart/docker-compose.yml

```diff
@@ -21,10 +21,17 @@ services:
     ports:
       - 9001:9001
       - 9000:9000
+    healthcheck:
+      test: [ "CMD", "curl", "--silent", "--fail", "--output", "/dev/null", "http://localhost:9000/minio/health/live" ]
+      start_period: 15s
+      interval: 5s
+      timeout: 10s
+      retries: 10
     command: ["server", "/data", "--console-address", ":9001"]
   mc:
     depends_on:
-      - minio
+      minio:
+        condition: service_healthy
```

Comment:
By default adding a dependency just ensures it is started - this waits to ensure the container is healthy before proceeding.